packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/mainTreeListMenu..st

```diff
@@ -0,0 +1,10 @@
+as yet unclassified
+mainTreeListMenu: aMenu
+	<treeListMenu>
+	
+	aMenu addStayUpItemSpecial.
```

Comment:
What does `addStayUpItemSpecial` do? For me, it doesn’t seem to make any difference at all. And to be honest, I didn’t really understand the method comment in `MenuMorph`…

Reply:
Same. We copied it for safety reasons. Quick test seemed okay without it. We removed it.